src/submission/submission-api.ts

```diff
@@ -1,26 +1,25 @@
 import * as submission from './submission-service';
+import * as persistedConfig from './persisted-config/service';
 import * as submission2Clinical from './submission-to-clinical/submission-to-clinical';
 import { Request, Response } from 'express';
-import { TsvUtils, ControllerUtils, isStringMatchRegex } from '../utils';
+import { TsvUtils, ControllerUtils } from '../utils';
 import { loggerFor } from '../logger';
 import {
   CreateRegistrationCommand,
   MultiClinicalSubmissionCommand,
   NewClinicalEntity,
   SubmissionBatchError,
   SubmissionBatchErrorTypes,
-  ClinicalEntityType,
-  BatchNameRegex,
 } from './submission-entities';
 import { HasFullWriteAccess, HasProgramWriteAccess } from '../auth-decorators';
-import jwt from 'jsonwebtoken';
 import _ from 'lodash';
 import { batchErrorMessage } from './submission-error-messages';
 const L = loggerFor(__filename);
 
 class SubmissionController {
   @HasProgramWriteAccess((req: Request) => req.params.programId)
   async getRegistrationByProgramId(req: Request, res: Response) {
+    if (await submissionSystemIsDisabled(res)) return;
```

Comment:
I'm not sure if the checks should be here

Reply:
I would rather put them in the submission service

Reply:
hmm, I thought about that too, but it has to be done in `submission-to-clinical` too because `commit` & `approve` are their. Also it didn't really make sense to do any processing if the system is disabled...